src/arrays.js

```diff
@@ -15,7 +19,17 @@ function sameFirst(a, b) {
  * @return {Boolean}
  */
 function deepEquals(a, b) {
-
+  for (i = 0; i < a.length; i++) {
+    if (a.length != b.length) {
+      return false;
+    }
+    if (a[i] === b[i]) {
```

Comment:
Instead of having an empty block for `if` and doing everything in `else`, just reverse things to check NOT equal.